chapter4.adoc

```diff
@@ -221,9 +220,54 @@ follows:
 ], config:{lanes: 1, hspace:1024}}
 ....
 
+=== MTT access permissions lookup process
+
+MTT access permissions for a physical address PA in the context of a
+supervisor domain is ascertained as follows:
+
+1. Let a be `mttp.ppn`x`PAGESIZE`, and let i = LEVELS, where for mode
+Smmtt[34][*], LEVELS = 2 and for Smmtt[46, 56][*rw*] LEVELS = 3; PAGESIZE is
+2^12; MTT NON_LEAF_PTE_SIZE = 8 bytes. The `mttp` register must be active,
```

Comment:
Any reason why we call it NON_LEAF_PTE_SIZE instead of MPTE_SIZE?

Reply:
at the leaf level, the entry size for MTT entries are 2 bits and 4 bits for Smmtt[*] and Smmtt[*]rw resp.